apps/src/lib/node/ledger/shell/vote_extensions.rs

```diff
@@ -238,9 +238,17 @@ mod extend_votes {
         vote_extensions.into_iter().filter_map(|vote| {
             SignedExt::try_from_slice(&vote.vote_extension[..])
                 .map_err(|err| {
-                    tracing::error!(
+                    tracing::warn!(
                         ?err,
-                        "Failed to deserialize signed vote extension",
+                        validator_tendermint_addr = ?vote.validator,
+                        vote_extension_len = vote.vote_extension.len(),
+                        "Failed to deserialize signed vote extension, will \
+                         ignore",
+                    );
+                    tracing::debug!(
+                        ?vote.vote_extension,
+                        validator_tendermint_addr = ?vote.validator,
+                        "Raw undeserializable vote extension for validator",
```

Comment:
same here

Reply:
:+1: mentioned Ethereum events in the logs here